magma/conversions.py

```diff
@@ -257,7 +260,8 @@ def _tuple(value, n=None, t=Tuple):
     if isinstance(value, t):
         return value
 
-    if not isinstance(value, (Digital, Array, IntegerTypes, Sequence, Mapping)):
+    if not isinstance(magma_value(value),
+                      (Digital, Array, IntegerTypes, Sequence, Mapping)):
```

Comment:
we should probably also change the error message here, like "bit() can only be used on Bit, Array, int, or wrapped protocol type, not {type(value)}"

Reply:
for now, leave it as is in this PR. wrapped protocol type is not enough, but the underlying type of the wrapped protocol type should be Bit, Array, or int, in this case.